db/upgrade-X.X.X-X.Y.Z.sql

```diff
@@ -1,3 +1,33 @@
 --
 -- PacketFence SQL schema upgrade from X.X.X to X.Y.Z
 --
+
+--
+-- Add a primary key to the radacct_log
+--
+
+ALTER TABLE `radacct_log` ADD COLUMN `radacct_log_id` int NOT NULL PRIMARY KEY AUTO_INCREMENT FIRST;
```

Comment:
I'd personally stick with the simple `id` as it tends to be the default standard for MySQL tables but if I'm the only one that likes it better with the short format, we can leave it as is